src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions.Internal/IUpgradeAssistantConfigurationLoader.cs

```diff
@@ -1,15 +1,12 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Threading;
-using System.Threading.Tasks;
-
 namespace Microsoft.DotNet.UpgradeAssistant
 {
     public interface IUpgradeAssistantConfigurationLoader
     {
-        Task SaveAsync(UpgradeAssistantConfiguration configuration, CancellationToken token);
+        UpgradeAssistantConfiguration Load();
 
-        Task<UpgradeAssistantConfiguration> LoadAsync(CancellationToken token);
+        void Save(UpgradeAssistantConfiguration configuration);
```

Comment:
Is this one of the instances that will be made async in the next iteration? Is there any perf hits in making this to be sync now?

Reply:
No, this will stay sync (maybe go back to async). However, the config file is small, so we can just load it into memory and then parse it via the sync patterns